drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/ScenarioSimulationEventHandler.java

```diff
@@ -208,24 +208,28 @@ public void onEvent(AppendColumnEvent event) {
         context.getStatus().setColumnId(String.valueOf(new Date().getTime()));
         context.getStatus().setColumnGroup(event.getColumnGroup());
         commonExecution(context, new AppendColumnCommand());
+        context.getScenarioGridPanel().setFocus(true);
```

Comment:
@jomarko 
If there is nothing preventing this, move all those "context.getScenarioGridPanel().setFocus(true);" inside commonExecution